ecommers/lib/ui/widgets/recently_viewed_widget/recently_viewed_list.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:ecommers/ui/decorations/dimens/index.dart';
+import 'package:ecommers/ui/widgets/recently_viewed_widget/index.dart';
+import 'package:flutter/cupertino.dart';
+
+class RcentlyViewedList extends StatelessWidget {
```

Comment:
please consider moving all content inside RecentlyViewedWidget
And RcentlyViewedList going to be unnecessary